lib/grandpa/message_tracker.go

```diff
@@ -19,25 +19,26 @@ type tracker struct {
 	blockState BlockState
 	handler    *MessageHandler
 	votes      votesTracker
-
-	// map of commit block hash to commit message
-	commitMessages map[common.Hash]*CommitMessage
-	mapLock        sync.Mutex
-	in             chan *types.Block // receive imported block from BlockState
-	stopped        chan struct{}
+	commits    commitsTracker
+	mapLock    sync.Mutex
+	in         chan *types.Block // receive imported block from BlockState
```

Comment:
`in` can mean many things, I believe that `importedBlocks` is more clear

Reply:
That's code that was there before and changes would be out of scope.
On top of your comment, I would add we should not use a channel and use methods instead (maybe with a mutex if we need). This is especially true when communicating between different interfaces.